### ISSUE TITLE
error docker pull, image tag doesnt match (docker&git tag 14 vs postgres version 15)

### ISSUE THREAD

**Issue by saulmmendoza** [sameersbn/docker-postgresql#164]
Hello!

First of all thank you for upgrading the postgres image 🐳 

I want to report that the current image tag doesnt match, the commit/PR #162 says the postgres version is 15 but the git tag is 14 and therefore the docker tag is also 14

## error
```sh
$ docker pull sameersbn/postgresql:15-20230628
Error response from daemon: manifest for sameersbn/postgresql:15-20230628 not found: manifest unknown: manifest unknown
```

## success
```sh
$ docker pull sameersbn/postgresql:14-20230628
14-20230628: Pulling from sameersbn/postgresql
6b851dcae6ca: Pull complete
2b4697fa2746: Pull complete
dddb44cdb2b7: Pull complete
a3151f07da42: Pull complete
c2bf59ad42b1: Pull complete
baf3436f3423: Pull complete
4f4fb700ef54: Pull complete
a91611886d34: Pull complete
Digest: sha256:3c04570aa3666584f35918afe72d7d27f6fcae08101d7aeb3fdc052caeda4590
Status: Downloaded newer image for sameersbn/postgresql:14-20230628
docker.io/sameersbn/postgresql:14-20230628
```

**Comment by sameersbn** [comment]
thanks for reporting. the image is now available for download

**Comment by danielsss** [comment]
@sameersbn Only `latest` tag is working now and all the other tags got the same error.

```shell
ubuntu@ip-172-31-30-168:~/ec2-applications$ docker pull sameersbn/postgresql:latest
latest: Pulling from sameersbn/postgresql
23884877105a: Already exists
bc38caa0f5b9: Already exists
2910811b6c42: Already exists
36505266dcc6: Already exists
d933cdea99ae: Pull complete
66bd7fe5752e: Pull complete
22cb7c408876: Pull complete
975306f62ae4: Pull complete
ea471c2b0d75: Pull complete
75e0820853f5: Pull complete
b6777cd7641d: Pull complete
Digest: sha256:73f2ea2b288dc6ce9b40ce5556d8f867c0d37783281114f0f2013b70fe05b643
Status: Downloaded newer image for sameersbn/postgresql:latest
docker.io/sameersbn/postgresql:latest
ubuntu@ip-172-31-30-168:~/ec2-applications$ docker pull sameersbn/postgresql:14-20230628
14-20230628: Pulling from sameersbn/postgresql
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'
ubuntu@ip-172-31-30-168:~/ec2-applications$ docker pull sameersbn/postgresql:15-20230628
15-20230628: Pulling from sameersbn/postgresql
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'
ubuntu@ip-172-31-30-168:~/ec2-applications$ docker pull sameersbn/postgresql:14-20230628
14-20230628: Pulling from sameersbn/postgresql
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'
ubuntu@ip-172-31-30-168:~/ec2-applications$
```

**Comment by dalang** [comment]
I need to upgrade my gitlab db container from sameersbn/postgresql:12-20200524 to sameersbn/postgresql:14-20230628, but I can not pull the sameersbn/postgresql:14-20230628 image.

the same error message as @danielsss 

> docker pull sameersbn/postgresql:15-20230628
15-20230628: Pulling from sameersbn/postgresql
mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'

---
thank danieless, the above issue had been resolved after upgrade my docker Version to 20.10.18.

**Comment by danielsss** [comment]
> I need to upgrade my gitlab db container from sameersbn/postgresql:12-20200524 to sameersbn/postgresql:14-20230628, but I can not pull the sameersbn/postgresql:14-20230628 image.
> 
> the same error message as @danielsss
> 
> > docker pull sameersbn/postgresql:15-20230628
> > 15-20230628: Pulling from sameersbn/postgresql
> > mediaType in manifest should be 'application/vnd.docker.distribution.manifest.v2+json' not 'application/vnd.oci.image.manifest.v1+json'

What's your docker version? It looks like that you need to upgrade the docker version first.
